templates/interstitial.html

```diff
@@ -1,10 +1,9 @@
-{% extends 'layouts/_questionnaire.html' %}
-{% from 'macros/helpers.html' import format_paragraphs %}
+{% extends "layouts/_questionnaire.html" %}
 
+{% from 'macros/helpers.html' import format_paragraphs %}
 {% from 'macros/helpers.html' import interviewer_note %}
```

Comment:
Its left single quotes here, shouldn't they be double?

Reply:
I found another 10+ issues like that, fixed now.